tests/Fixer/ClassNotation/NoUnneededFinalMethodFixerTest.php

```diff
@@ -51,6 +51,16 @@ private function baz() {}
     final private function baz() {}
 }',
             ],
+            'private-method' => [
+                '<?php
+class Foo {
+    private function baz() {}
+}',
+                '<?php
+class Foo {
+    final private function baz() {}
```

Comment:
`final private` method. kudos for PHP

Reply:
Hehe, actually had an example of this lying around somewhere in **my** code, that's how I figured I should take a look.